api/all/src/test/java/io/opentelemetry/api/trace/propagation/W3CTraceContextPropagatorTest.java

```diff
@@ -450,8 +449,8 @@ void fieldsList() {
 
   @Test
   void headerNames() {
-    Assertions.assertThat(W3CTraceContextPropagator.TRACE_PARENT).isEqualTo("traceparent");
-    Assertions.assertThat(W3CTraceContextPropagator.TRACE_STATE).isEqualTo("tracestate");
+    assertThat(W3CTraceContextPropagator.TRACE_PARENT).isEqualTo("traceparent");
```

Comment:
not sure why this changed leaked into this PR, but I also don't care too much, because it's just in a test and a good change.